deploy/coins/bitcoin/scripts/entry-dev.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
+RPC_USER="${RPC_USER:=serai}"
+RPC_PASS="${RPC_PASS:=seraidex}"
+
+# address: bcrt1q7kc7tm3a4qljpw4gg5w73cgya6g9nfydtessgs
+# private key: cV9X6E3J9jq7R1XR8uPED2JqFxqcd6KrC8XWPy1GchZj7MA7G9Wx
+MINER="${MINER:=bcrt1q7kc7tm3a4qljpw4gg5w73cgya6g9nfydtessgs}"
+PRIV_KEY="${PRIV_KEY:=cV9X6E3J9jq7R1XR8uPED2JqFxqcd6KrC8XWPy1GchZj7MA7G9Wx}"
+BLOCK_TIME=${BLOCK_TIME:=5}
+
+bitcoind -regtest -txindex -fallbackfee=0.000001 -rpcuser=$RPC_USER -rpcpassword=$RPC_PASS -rpcallowip=0.0.0.0/0 -rpcbind=127.0.0.1 -rpcbind=$(hostname) &
+
+# give time to bitcoind to start
+while true
+do
+	bitcoin-cli -regtest -rpcuser=$RPC_USER -rpcpassword=$RPC_PASS generatetoaddress 100 $MINER && break
+	sleep 5
```

Comment:
When I pull up this raw file, it claims it's a one space ident? Is that correct?